autoload/miniSnip.vim

```diff
@@ -62,7 +62,13 @@ function! s:directories() abort
 endfunction
 
 function! s:findSnippetFile() abort
```

Comment:
This isn't the only place where the `\f` pattern is used.  
It's also used in `s:insertSnippet()`, `miniSnip#completeFunc()` and `miniSnip#completeMapping()`

Reply:
Corrected. Take a look.
I need to do more test and changes in docs.